apps/allocations/app/components/App/Budgets.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+import { Viewport } from '@aragon/ui'
+
+import { CARD_STRETCH_BREAKPOINT } from '../../utils/responsive'
+import { Budget } from '../Card'
+
+const Budgets = ({
+  budgets,
+  onNewAllocation,
+  onEdit,
+  onDeactivate,
+  onReactivate,
+}) => {
+  if (!budgets) return
+
+  return (
+    <Viewport>
+      {({ width }) => {
+        const screenSize = width
+
+        return (
+          <StyledAccounts screenSize={screenSize}>
+            {budgets.map(({ budgetId, data }) => (
+              <Budget
+                key={budgetId}
```

Comment:
Could we have a better `key` alternative mechanism like using `uuids`? Using the same `budgetId` could make this brittle in some cases

Reply:
Do we really need this? In what scenarios are `budgetId`s going to be duplicated? Maybe it should be a matter of ensuring upstream that they are unique.

Reply:
Is more related to re-renders, in the case an element gets removed and another takes the same id, it could cause some issues with React reconciliation, but as you point it is probably an unlikely scenario for this specific case and more a good practice to adopt :)